content/en/docs/tasks/job/fine-parallel-processing-work-queue.md

```diff
@@ -59,6 +59,13 @@ You could also download the following files directly:
 - [`rediswq.py`](/examples/application/job/redis/rediswq.py)
 - [`worker.py`](/examples/application/job/redis/worker.py)
 
+To start a single instance of Redis, you need to create the redis pod and redis service:
+
+```shell
+# this assumes you downloaded the manifests already
+kubectl apply -f ./redis-pod.yaml
+kubectl apply -f ./redis-service.yaml
```

Comment:
Even better: use the HTTP URLs of those manifests (and even even better, change this page to do that throughout).

See https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#creating-a-statefulset for an example of a page that uses URLs for manifests.

Reply:
Done, thanks for the suggestion :)

Note that I left the other instances untouched on this page, as they require manual edits.